### ISSUE TITLE
Mention SQL-92

### ISSUE THREAD

**Issue by sfkeller** [partiql/partiql-spec#7]
Spec. or at least the announcement blog post says PartiQL is "maintaining strict backwards compatibility with SQL".
So I'd suggert to mention "SQL-92" (or newer?) in the spec. and elsewhere.

**Comment by almann** [comment]
Good point, I definitely think we should be clearer about what levels of SQL compliance in the spec itself.
